ansible/roles/export_client_to_controller/tasks/main.yaml

```diff
@@ -0,0 +1,33 @@
+# This role only works correctly on localhost due to the use of fetch
+# It creates ~/ostest-working initially containing kubeconfig, and put oc in
+# ~/.local/bin/
+
+# Also used indirectly by default variables
+- name: Include variables
+  include_vars: vars/default.yaml
+
+# We use mkdir instead of file below because the local shell always expands ~
+# correctly locally, and doesn't try to change the permissions on the local
+# directory to the remote user
+
+- name: Ensure local directories exist
+  shell: |
+    mkdir -p {{ local_working_dir }}
```

Comment:
where is local_working_dir defined?

Reply:
Incidentally, I can't remember if I left a comment in, but the mkdir in shell works around ansible weirdness with connection: local. I'd like to think there's a better way to do this in ansible, but this the best I came up with.